rocky/reports/views/aggregate_report.py

```diff
@@ -95,13 +96,16 @@ def setup(self, request, *args, **kwargs):
         )
 
     def get(self, request, *args, **kwargs):
-        if not self.selected_oois:
+        if not self.selected_oois and "all" not in self.selected_oois:
```

Comment:
This seems redundant. If self.selected_oois is empty, it can also not contain 'all'